internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -82,89 +88,212 @@ var Manifest = &appsv1.DaemonSet{
 					},
-					SecurityContext: &v1.SecurityContext{
-						AllowPrivilegeEscalation: &falsely,
-						ReadOnlyRootFilesystem:   &truly,
-						Capabilities: &v1.Capabilities{
-							Drop: []v1.Capability{"ALL"},
-						},
-						RunAsUser:  &userRootless,
-						RunAsGroup: &userRootless,
-						SELinuxOptions: &v1.SELinuxOptions{
-							// TODO(jaosorior): Use a more restricted selinux type
-							Type: "spc_t",
+				},
+				Containers: []v1.Container{
+					{
+						Name:            config.OperatorName,
+						Args:            []string{"daemon"},
+						ImagePullPolicy: v1.PullAlways,
+						VolumeMounts: []v1.VolumeMount{
+							{
+								Name:      "host-operator-volume",
+								MountPath: config.ProfilesRootPath,
+							},
+							{
+								Name:      "selinux-drop-dir",
+								MountPath: SelinuxDropDirectory,
+							},
+							{
+								Name:      "selinuxd-socket-volume",
+								MountPath: SelinuxdSocketDir,
+							},
 						},
-					},
-					Resources: v1.ResourceRequirements{
-						Requests: v1.ResourceList{
-							v1.ResourceMemory:           resource.MustParse("64Mi"),
-							v1.ResourceCPU:              resource.MustParse("100m"),
-							v1.ResourceEphemeralStorage: resource.MustParse("50Mi"),
+						SecurityContext: &v1.SecurityContext{
+							AllowPrivilegeEscalation: &falsely,
+							ReadOnlyRootFilesystem:   &truly,
+							Capabilities: &v1.Capabilities{
+								Drop: []v1.Capability{"ALL"},
+							},
+							RunAsUser:  &userRootless,
+							RunAsGroup: &userRootless,
+							SELinuxOptions: &v1.SELinuxOptions{
+								// TODO(jaosorior): Use a more restricted selinux type
+								Type: "spc_t",
+							},
 						},
-						Limits: v1.ResourceList{
-							v1.ResourceMemory:           resource.MustParse("128Mi"),
-							v1.ResourceCPU:              resource.MustParse("300m"),
-							v1.ResourceEphemeralStorage: resource.MustParse("200Mi"),
+						Resources: v1.ResourceRequirements{
+							Requests: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("64Mi"),
+								v1.ResourceCPU:              resource.MustParse("100m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("50Mi"),
+							},
+							Limits: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("128Mi"),
+								v1.ResourceCPU:              resource.MustParse("300m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("200Mi"),
+							},
 						},
-					},
-					Env: []v1.EnvVar{
-						{
-							Name: config.NodeNameEnvKey,
-							ValueFrom: &v1.EnvVarSource{
-								FieldRef: &v1.ObjectFieldSelector{
-									FieldPath: "spec.nodeName",
+						Env: []v1.EnvVar{
+							{
+								Name: config.NodeNameEnvKey,
+								ValueFrom: &v1.EnvVarSource{
+									FieldRef: &v1.ObjectFieldSelector{
+										FieldPath: "spec.nodeName",
+									},
 								},
 							},
 						},
 					},
-				}},
+					{
+						Name:  "selinuxd",
+						Image: "quay.io/jhrozek/selinuxd",
```

Comment:
let's switch to `jaosorior/selinuxd` as that's what's currently being built automatically when we push to the selinuxd repo. We should now add a github issue about moving that to an appropriate org. (First we gotta figure what that org is...)

Reply:
Oops, sorry. This was a leftover from when I was testing some custom image of my own.

Reply:
Fixed.